### PR TITLE
gql_compose

### PR DESCRIPTION
**Pull request by asmab89**

### CODE REVIEW

File: e2e-tests/spring-graphql/src/main/kotlin/com/foo/graphql/allLimitReached/DataRepository.kt [hunk 1]
```diff
@@ -0,0 +1,52 @@
+package com.foo.graphql.allLimitReached
```
Comment (arcuri82) [inline e2e-tests/spring-graphql/src/main/kotlin/com/foo/graphql/allLimitReached/DataRepository.kt:1]: should avoid upper-case letter in the name of packages. see:
https://github.com/EMResearch/EvoMaster/blob/master/docs/for_developers.md#naming-convention

**Review by arcuri82 — CHANGES_REQUESTED** [review]
recall to pull from `master`

File: core/src/main/kotlin/org/evomaster/core/problem/graphql/builder/GraphQLActionBuilder.kt [hunk 2]
```diff
@@ -130,9 +130,23 @@ object GraphQLActionBuilder {
...
+        if (params.any { p -> p is GQReturnParam }) {
+
+            if (!params.find { p -> p is GQReturnParam }?.let { isAllLimitInObjectFields(it) }!!  ) {
+                val action = GraphQLAction(actionId, element.fieldName, type, params)
+                actionCluster[action.getName()] = action
+            }
+
+        }else {
+            val action = GraphQLAction(actionId, element.fieldName, type, params)
+            actionCluster[action.getName()] = action
```
[the remark this answers is not in the document]
Comment (asmab89) [reply core/src/main/kotlin/org/evomaster/core/problem/graphql/builder/GraphQLActionBuilder.kt]: Done

File: e2e-tests/spring-graphql/src/main/kotlin/com/foo/graphql/allLimitReached/DataRepository.kt (hunk 1 above)
Comment (asmab89) [reply e2e-tests/spring-graphql/src/main/kotlin/com/foo/graphql/allLimitReached/DataRepository.kt]: Done

**Comment by arcuri82** [comment]
hi @asmab89  did u see the comment `recall to pull from master` ? (should fix the issue with E2E failing in JDK 8)

**Pull request merged**